database/deployment/list.go

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package deployment
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/go-vela/types/constants"
+	"github.com/go-vela/types/database"
+	"github.com/go-vela/types/library"
+)
+
+// ListDeployments gets a list of all deployments from the database.
+func (e *engine) ListDeployments(ctx context.Context) ([]*library.Deployment, error) {
+	e.logger.Trace("listing all deployments from the database")
+
+	// variables to store query results and return value
+	d := new([]database.Deployment)
+	deployments := []*library.Deployment{}
+
+	// COUNT DB
+
+	// COUNT GH
+
+	//
```

Comment:
whats up here?

Reply:
unsure, I think I did that a few months ago lol but then realized I didn't need to do anything there